code/modules/client/client_procs.dm

```diff
@@ -374,7 +374,7 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
 		winset(src, "[topmenu.type]", "parent=menu;name=[url_encode(topmenuname)]")
```

Comment:
Nevermind what I said, The real fix is to add a `url_encode` to `topmenu.type`. at this line.

Reply:
hmmmmmmmmmmmmmm. wtf byond